cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
 
+// const { should } = require("chai");
+// import { should } from 'chai';
+import { user } from '../support/testDate';
+
 describe('Book Store app', () => {
   before(() => {
+    cy.visit('/login');
     
   });
 
-  it('', () => {
-    
+  it('login', () => {
+    cy.get('#userName').type(user.username);
+    cy.get('#password').type(user.password);
+    cy.get('#login').click();
+    cy.get('.form-label').should('contain.text', 'iziumova');
+    cy.url().should('eq', 'https://demoqa.com/profile');
+  
+
   });
+
+  it('add book', () => {
+    cy.login();
+    cy.visit('/profile');
```

Comment:
Remove the `cy.visit('/profile')` from the 'add book' test as the login command should redirect to the profile page.